README.md

```diff
@@ -1,15 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+  LEADERBOARD
```

Comment:
- [ ] Please replace `# Project Name` with `# Leaderboard` because your project name should be your heading :+1: